src/app/pages/virtualization/components/all-instances/instance-details/instance-general-info/instance-edit-form/instance-edit-form.component.ts

```diff
@@ -58,10 +60,15 @@ export class InstanceEditFormComponent {
   title: string;
   editingInstance: VirtualizationInstance;
 
+  get isVmInstanceType(): boolean {
```

Comment:
I've updated master with another way to determine if it's a VM based on Denyses PR.